rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date)
```

Comment:
- [ ] You should also pass the book and person ad parameters to the initialize method that way you can create a rental with `Rental.new(...etc)`  and also achieve the many to many relationships between person and rental 
```suggestion
  def initialize(date, book, person)
```